### ISSUE TITLE
Firmware Update stuck at 0x3fc (1.8.1)

### ISSUE THREAD

**Issue by njordan77** [arduino-libraries/UnoWiFi-FirmwareUpdater-Plugin#3]
Any idea what i can do now.....firmware update stuck at 0x3fc.....no further progress...no matter which firmware I try out. Using IDE 1.81.....Regards. Norbert

**Comment by chrisTactition** [comment]
I see the same issue using 1.8.1 on a Mac. The firmware update tool quickly reached "Writing 4096@0x3fc..." but then froze and has made no more progress.

**Comment by njordan77** [comment]
for me finally it worked,....., the reason is that I was too impatient. for whatever reason it is stuck at 3fc but after several minutes it continued successfully. I know ESPs very well and normally flashing behaves differently....whatever, at least its working again. anyhow i'm not sure what the Dec'update improves over the March'16 firmware....

**Comment by chrisTactition** [comment]
After your recommendation to wait longer I was also able to program it successfully. There should be a message to warn people that the process may take a few minutes. Mine took about 5 minutes on that step and I assumed it failed.

**Comment by sergiotomasello** [comment]
yes, it's a long time process due to low baud rate between 328P and ESP8266.
